crawler/src/main/java/com/movie/recommender/crawler/service/MongoDBService.java

```diff
@@ -0,0 +1,29 @@
+package com.movie.recommender.crawler.service;
+import com.mongodb.MongoException;
+import com.movie.recommender.common.model.movie.MovieDetails;
+import com.movie.recommender.crawler.repository.MovieRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@Slf4j
+public class MongoDBService {
```

Comment:
add an `isDataInitialized` method to the service to check if data has already been initialized